### ISSUE TITLE
issue when merge to android x

### ISSUE THREAD

**Issue by sherif-farid** [getbouncer/cardscan-android#40]
please update dependencies to androidX

**Comment by kingst** [comment]
Which version of the library are you using? We've tested it with apps that use androidx and it doesn't have any problems. Can you provide an example that shows the issue?

**Comment by eddiecrawford** [comment]
I'm experiencing the same thing with the latest `1.0.5105` release.

IE: 
```
> Task :ibotta-app:mergeDebugNativeLibs FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':ibotta-app:mergeDebugNativeLibs'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > More than one file was found with OS independent path 'lib/x86_64/librsjni_androidx.so'
```

After adding these to my gradle's `android {}` space:
```
    packagingOptions {
        pickFirst 'lib/**/libRSSupport.so'
        pickFirst 'lib/**/librsjni.so'
        pickFirst 'lib/**/librsjni_androidx.so'
    }
```
It compiles... However, I don't like having to rely on that solution 😬

**Comment by kingst** [comment]
Ok, I'll look into this. I'll either have the conversion in place this week or an answer as to why it will take longer.

**Comment by kingst** [comment]
It's coming -- we'll do a deploy later today https://github.com/getbouncer/cardscan-android/pull/48

**Comment by kingst** [comment]
Merged and deployed, version `1.0.5106` includes AndroidX support, closing issue

**Comment by eddiecrawford** [comment]
Solid! Awesome turnaround on this. 
Thank you 🙏
